src/main/java/com/google/api/generator/gapic/protoparser/PluginArgumentParser.java

```diff
@@ -89,8 +94,8 @@ private static Optional<String> parseConfigArgument(String pluginProtocArgument,
   }
 
   @VisibleForTesting
-  static boolean hasMetadataFlag(String pluginProtocArgument) {
-    return Arrays.stream(pluginProtocArgument.split(COMMA)).anyMatch(s -> s.equals(KEY_METADATA));
+  static boolean hasFlag(String pluginProtocArgument, String flagKey) {
+    return Arrays.asList(pluginProtocArgument.split(COMMA)).contains(flagKey);
```

Comment:
I believe stream should be preferred way, as it does not create actual list and things happen on the fly (like one pass and no intermediate list, unlike two passes (one to create list and another one to do contains()).

Reply:
Reverted to stream.